### PR TITLE
Feature/essifi 101

### PR DESCRIPTION
**Pull request by maikel-maas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+
+```shell
+# PORT - defaults to 3000
+export PORT=5000
+
+# ENV - defaults to development
+export ENV=prod
+```
+
+## Docker
```
Comment (hrehman-sphereon) [inline README.md:49]: Perhaps we can add the line fromPR of ESSIFI-102

File: README.md [hunk 2]
```diff
@@ -1 +1,49 @@
...
+## Environment variables
+
+There are two environment variables that may be set
+
+```shell
+# PORT - defaults to 3000
+export PORT=5000
+
+# ENV - defaults to development
+export ENV=prod
```
Comment (hrehman-sphereon) [inline README.md:46]: We can highlight that it is optional

File: README.md [hunk 3]
```diff
@@ -1 +1,49 @@
...
+## Active Development
+_IMPORTANT: This software still is in early development stage. As such you should expect breaking changes in APIs, we expect to keep that to a minimum though._
+
+## Background
+
+The pe-state-machine project aims to implement the PE-OPENAPI and use PE-JS library and Veramo API as a dependency. 
+
+## How to run it
+
+From the root folder:
```
Comment (hrehman-sphereon) [inline README.md:19]: Perhaps we can add DB creation command for mongo

File: README.md [hunk 4]
```diff
@@ -1 +1,49 @@
...
+node dist/src/index.js
+
+# running it via npm
+npm install
+
+npm run build
+
+node dist/src/index.js
+```
+
```
Comment (hrehman-sphereon) [inline README.md:36]: Perhaps we can add more information about the POSTMAN workspace.

**Review by hrehman-sphereon — CHANGES_REQUESTED** [review]
Made some small comments. Perhaps we can have a look at those to test it better and burn the ticket ?

File: README.md (hunk 1 above)
Comment (maikel-maas) [reply README.md:49]: That PR had some problems, unneeded files in the image, the script wasn't working. It needs to be refactored.

File: README.md (hunk 3 above)
Comment (maikel-maas) [reply README.md:19]: There is no such thing. The database is created on the first insertion.

File: README.md (hunk 4 above)
Comment (maikel-maas) [reply README.md:46]: Done!

**Review by hrehman-sphereon — APPROVED** [review]
Entries can be saved in the Mongo. And the postMan collection is usable. Approved.

**Pull request merged**